server/api/scheduler.go

```diff
@@ -74,6 +74,20 @@ func (h *schedulerHandler) List(w http.ResponseWriter, r *http.Request) {
 		}
 		h.r.JSON(w, http.StatusOK, pausedSchedulers)
 		return
+	case "disabled":
```

Comment:
Could you add a correspond command in `pd-ctl` ?

Reply:
ok

Reply:
Done. PTAL.  Sorry for the slow response :)